retreats/2025.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: event
+title: Suspension Retreat 2025
+image: venues/studio_friction.jpg
+event_id: retreat2025
```

Comment:
Continuing from comment above:

So this maps to `_data/retreat2025/...`.

This allows use to have structured data in yaml files in `_data/retreat2025/` and makes all of this reusable and easier to work with.